modules/sms_email_reminder/cron_functions.php

```diff
@@ -1,148 +1,71 @@
-
-        $cnt = "";
-        $cnt .= "\nHeaders : " . $headers;
-        $cnt .= "\nDate Time :" . date("d M, Y  h:i:s");
-        $cnt .= "\nTo : " . $to;
-        $cnt .= "\nSubject : " . $subject;
-        $cnt .= "\nBody : \n" . $vBody . "\n";
-
-        if (1) {
-            //WriteLog($cnt);
-        }
-
+    if ($GLOBALS['SMTP_HOST'] == '') {
         $mstatus = true;
-        $mstatus = @mail($to, $subject, $vBody, $headers);
-        // larry :: debug
-        //echo "\nDEBUG :email: send email from=".$from." to=".$to." sbj=".$subject." body=".$vBody." head=".$headers."\n";
-        //echo "\nDEBUG :email: send status=".$mstatus."\n";
+        $mstatus = @mail($to, $cc, $subject, $vBody);
     } else {
-        // larry :: debug
-        //echo "\nDEBUG :: use smtp method\n";
-
-        if (!class_exists("smtp_class")) {
-            include("../../library/classes/smtp/smtp.php");
-            include("../../library/classes/smtp/sasl.php");
-        }
-
-        $strFrom = $from;
-        $sender_line = __LINE__;
-        $strTo = $to;
-        $recipient_line = __LINE__;
-        if (strlen($strFrom) == 0) {
-            return( false );
-        }
-
-        if (strlen($strTo) == 0) {
-            return( false );
-        }
-
-        //if( !$smtp )
-        $smtp = new smtp_class();
-
-        $smtp->host_name = $GLOBALS['smtp_host_name'];
-        $smtp->host_port = $GLOBALS['smtp_host_port'];
-        $smtp->ssl = $GLOBALS['smtp_use_ssl'];
-        $smtp->localhost = $GLOBALS['smtp_localhost'];
-        $smtp->direct_delivery = 0;
-        $smtp->timeout = 10;
-        $smtp->data_timeout = 0;
-
-        $smtp->debug = 1;
-        $smtp->html_debug = 0;
-        $smtp->pop3_auth_host = "";
-
-        $smtp->user = $GLOBALS['smtp_auth_user'];
-        $smtp->password = $GLOBALS['smtp_auth_pass'];
-
-        $smtp->realm = "";
-        // Workstation name for NTLM authentication
-        $smtp->workstation = "";
-        // Specify a SASL authentication method like LOGIN, PLAIN, CRAM-MD5, NTLM, etc..
-        // Leave it empty to make the class negotiate if necessary
-        $smtp->authentication_mechanism = "";
-
-        // If you need to use the direct delivery mode and this is running under
-        // Windows or any other platform
-        if ($smtp->direct_delivery) {
-            if (!function_exists("GetMXRR")) {
-                $_NAMESERVERS = array();
-                include("getmxrr.php");
-            }
+            $SenderName = $GLOBALS['patient_reminder_sender_name'];
+            $SenderEmail = $GLOBALS['patient_reminder_sender_email'];
         }
-
-        if (
-            $smtp->SendMessage(
-                $strFrom,
-                array( $strTo ),
-                array(
-                "From: $strFrom",
-                "To: $strTo",
-                "Subject: $subject",
-                "Date Time :" . date("d M, Y  h:i:s")
-                ),
-                $vBody
-            )
-        ) {
+    {
```

Comment:
- remove lines 39 and 40 (those brackets are not needed)
- then fix indentation in lines 37 and 38

Reply:
Ok, Thank You